content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -102,7 +102,7 @@ kubectl create secret docker-registry regcred --docker-server=<your-registry-ser
 where:
 
 * `<your-registry-server>` is your Private Docker Registry FQDN.
-  Use `https://index.docker.io/v2/` for DockerHub.
+  Use `https://index.docker.io/v1/` for DockerHub.
```

Comment:
Looks like v1 has been removed. "https://index.docker.io/v1" returns 404 now.

Reply:
Thanks @tenqm for looking at this.

But on running `kubectl create secret docker-registry --help` it shows the following under Options : 
`--docker-server='https://index.docker.io/v1/': Server location for Docker registry`

Reply:
alright, then the kubectl help should be changed. Pls propose a fix to upstream community.

Reply:
Thanks, @tengqm 
Sure, I will propose a fix.

Reply:
Hi @tengqm, before i will raise a PR i have some observations please have a look.
when I run docker login then it takes default  https://index.docker.io/v1/ as a server value. I tried it on docker version 20.10.7.  below is the example of a config file:
```
{      
  "auths": {
                "https://index.docker.io/v1/": {
                        "auth": ".....4"
                }
}
```
same in [docs](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#log-in-to-docker)
when I run the `docker info` command:
:
`Registry: https://index.docker.io/v1/`
:
By default, Docker uses v1 API, if v1 is deprecated then it will use v2.

Is it legit to change in upstream k/k? what do you suggest